solutions/cf-1468D.mdx

```diff
@@ -0,0 +1,104 @@
+---
+id: cf-1468D
+source: CF
+title: Firecrackers
+author: Jesse Choe
+---
+
+[Official Editorial](https://assets.codeforces.com/rounds/1468/tutorials.pdf)
```

Comment:
would it be nicer to link to the [blog](https://codeforces.com/blog/entry/85951) directly?